.dockerignore

```diff
@@ -0,0 +1,25 @@
+**/.classpath
+**/.dockerignore
```

Comment:
we are ignoring **/.dockerignore but yet committing it

Reply:
Good shout, will un-commit

Reply:
Although we should consider keeping a copy in the repo so when new people download it they don't have to create a new one?

Reply:
I would keep the .dockerignore in root as it's an important file, otherwise we might fall back into the 'it works on my machine' due to people having different dockerignores overtime

Reply:
I think if it's a repo wide ignore it should live in the git ignore, docker ignore makes more sense for when we want to ignore docker related files for composing.